### DISCUSSION TITLE
Release SDK1.13 production date

### DISCUSSION THREAD

**Discussion by kletersteegp** [OP-TED/eForms-SDK#1076] [Q&A] [answered]
Hello,
We are almost ready implementing SDK1.13.
Is there a reason why release SDK1.13 only becomes available in Februari 2025? 
In the past there was not such a big time gap between the release of a SDK and becoming available in production.
We would really welcome it when SDK1,13 will becomer sooner available in production.

Regards,
Peter van de Kletersteeg.

**Comment by kletersteegp** [comment]
As I understood from the TED Together meeting: Is SDK1.13 already available in production for users that only use the API to submit notices?

**Comment by ferrakaop** [comment, marked as answer]
The reason that SDK 1.13 will be activated in February 2025 and not earlier is due to the need to implement all the changes introduced by the 2nd amendment of the eForms Implementing Regulation that are included in this SDK release.

The TED website needs to support the 6 new voluntary forms and all the other changes (new BTs, new NUTS, etc) and all these require significant effort in order to be implemented properly (including a series of technical activities that are quite heavy e.g. full re-indexation of its content).  Our contractor can only make these changes in a serialised  way and extremely carefully in order to avoid regressions.

In general we aim to keep the time gap to a minimum between the release of an SDK version and its activation in Production but as the TED website needs to handle every aspect of SDK 1.13 (we cannot choose which voluntary features to implement), it wasn’t possible to go any faster this time.

In the meantime, all the API functions in Preview can handle SDK 1.13 (submission, validation, visualisation) so it's possible to test all eSender needs.  In Production, it is possible to use the visualisation API with SDK 1.13 notices because it automatically adapts to the SDK versions published on Maven - but it will behave identically to the visualisation in Preview.  Submission and validation in Production need to wait until February, when the TED website can handle SDK 1.13 notices.
